src/DynamoCoreWpf/UI/Prompts/UsageReportingAgreementPrompt.xaml

```diff
@@ -4,6 +4,8 @@
         xmlns:ui="clr-namespace:Dynamo.UI"
         xmlns:p="clr-namespace:Dynamo.Wpf.Properties;assembly=DynamoCoreWpf"
         xmlns:views="clr-namespace:Dynamo.UI.Views"
+        Title="Agreements for Data Collection"
+        Name="AgreementsForDataCollection"
```

Comment:
Can we use a resource string instead...for localization?

Reply:
I need the `Name` for this window to use as unique id in automated tests. This window doesn't have one. I think we can localize `Title`, but I have never done this before, can you please give me an example?

Reply:
Here is an example: https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Debug/DebugModesWindow.xaml#L5